src/components/UserInfo.js

```diff
@@ -11,8 +12,14 @@ export default class UserInfo {
     };
   }
 
-  setUserInfo(userData) {
-    this._nameEl.textContent = userData.title;
-    this._jobEl.textContent = userData.description;
+  setUserInfo({ name, about, avatar, _id }) {
+    this._nameEl.textContent = name;
+    this._jobEl.textContent = about;
+    this._avatarEl.src = avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```

Reply:
I added this fix but makes the Profile Image black, I'm not sure where the problem is at.  Does it maybe have to do with the id?